api/v1beta1/kogitoruntime_types.go

```diff
@@ -15,6 +15,7 @@
 package v1beta1
 
 import (
+	"github.com/kiegroup/kogito-cloud-operator/api"
```

Comment:
If this implementation is the same as `v1`, you can inherit it here (so the CRD will be correctly generated). No problem having `v1beta1` to depend on `v1`.

Reply:
currently we don't have `v1`